### DISCUSSION TITLE
Audio delay when adding silent or existing core to Atmos TrueHD

### DISCUSSION THREAD

**Discussion by snuggglebun** [R3S3t9999/DoVi_Scripts#24] [Q&A] [answered]
Hi, love the scripts, I use them all the time and they have been very helpful. Lately, for some reason I cannot figure out, whenever I add a silent or existing core to an Atmos TrueHD (in order to mux it into a TS) they is a delay on audio. Sometimes it is a few seconds but sometimes it is several minutes. This occurs both when using Mode (4) to turn an ST DL into a DT DL and when using the standalone audio conversion in mode (8) and then muxing the result with the ST DL HEVC stream into a TS. I can post any log or anything you may need to reference, thank you advance!

**Comment by R3S3t9999** [comment]
Hi,
Please provide a small sample. You can make one with mkvtoolnix. 
I've remuxed tons of ST-DL mkv to DT-DL TS with workflow 4-3 for my x800m2 and never had sync issues even when the input has a delay...

**Reply by snuggglebun** [reply]
edit removed bad link

**Reply by R3S3t9999** [reply]
sorry, the link didn't work. Please upload to a traditional website like googledrive, dropbox, or MEGA.
https://mega.nz

**Reply by R3S3t9999** [reply, marked as answer]
Well, your sample is already out of sync...

This looks like Godzilla MO and I just tried with the original disc straight in tsmuxer and there's no sync issue with the DT-DL TS...
The disc has a 7007MS audio delay though and If you mux the disc to MKV, strangely the delay reported by mediainfo is negative 751 which is uncommon for the MKV container and would certainly confuse the script. So as I said, just mux it straight from the disc to TS and it will be fine.
